tools/experiments/experiments.js

```diff
@@ -290,6 +293,8 @@ function isExperimentOn_(id) {
       return [AMP_OPT_IN_COOKIE.BETA, _LEGACY_AMP_OPT_IN_COOKIE.BETA].includes(
```

Comment:
Curious: Can the legacy cookie stuff go away?

Reply:
Yep, done!